### DISCUSSION TITLE
Import all .js files from directory and map them to window

### DISCUSSION THREAD

**Discussion by james0r** [parcel-bundler/parcel#6181] [Q&A]
I'm using Alpine.js in my project. 

I want to have a components directory with each component being a JS file and in my app.js i want to map them to window[namespace].components.<component>

I achieved this with some help in a Webpack based project here --> [https://github.com/uicrooks/shopify-theme-lab/discussions/60#discussioncomment-502779](https://github.com/uicrooks/shopify-theme-lab/discussions/60#discussioncomment-502779)

How would I achieve this in ParcelJS?

Thank you.

**Comment by james0r** [comment]
Ended up switching to Webpack. Still interested to see how this is done in ParcelJS though.

**Comment by mischnic** [comment]
This could probably be solved with https://github.com/parcel-bundler/parcel/issues/4683
